### PR TITLE
Point URLs to localized guides

### PR DESCRIPTION
**Pull request by andrrms**
This guide is in Spanish, so links should point to Spanish guides too.

### CODE REVIEW

File: site/docs/es/plugins/emoji.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ import { EmojiFlavor, emojiParser } from "@grammyjs/emoji";
 
 // Esto se llama Context Flavoring
 // Puedes leer más en:
-// https://grammy.dev/guide/context.md#transformative-context-flavors
+// https://grammy.dev/es/guide/context.html#transformative-context-flavors
```
Comment (KnorpelSenf) [inline site/docs/es/plugins/emoji.md:39]: The anchor will not work, the section is named differently in Spanish.

File: site/docs/es/plugins/emoji.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ importar {
...
```
Comment (KnorpelSenf) [inline site/docs/es/plugins/emoji.md:71]: The anchor will not work, the section is named differently in Spanish.

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]

File: site/docs/es/plugins/emoji.md (hunk 1 above)
Comment (andrrms) [reply site/docs/es/plugins/emoji.md:39]: Actually I took the url anchor directly from the translated site. You should [check it](https://grammy.dev/es/guide/context.html#transformative-context-flavors) yourself (and on [GitHub](https://github.com/grammyjs/website/blob/main/site/docs/es/guide/context.md?plain=1#L377)).

**Review by KnorpelSenf — APPROVED** [review]
Thanks!

**Review by habemuscode — APPROVED** [review]
Looks nice!

**Review by roj1512 — APPROVED** [review]

**Pull request merged**